source/scss/_pf-basic-variables.scss

```diff
@@ -25,7 +25,9 @@
 // PatternFly box shadows
 // TODO: See what kind of shadows we need and define standards
 $pf-box-shadow: 0 .1em .2em rgba($color-pf-black-1000, 0.1);
-
+$pf-box-shadow-md: 0 .1em .3em rgba($color-pf-black-1000, 0.3);
```

Comment:
oh no, I forgot to delete the comments below (these were the pf3 values). 

Also, I'm not sure what variable should be included for defining this box-shadow. Should it be generic like I have here? Or should I create variables specific to the card for the hover and non-hover box shadows?